README.md

```diff
@@ -1,12 +1,26 @@
 ## ALiAS: Amity Linux Assistance Sapience
-- The following repo contains the source code for the official website of Amity Linux Assistance Sapience. 
+
+ [![HitCount](http://hits.dwyl.io/asetalias/asetalias.github.io.svg)](http://hits.dwyl.io/asetalias/asetalias.github.io) [![GitHub license](https://img.shields.io/github/license/asetalias/blog.svg?style=flat-square)](https://github.com/asetalias/asetalias.github.io/blob/master/LICENSE) 
```

Comment:
Why 'GitHub' License?
```suggestion
 [![HitCount](http://hits.dwyl.io/asetalias/asetalias.github.io.svg)](http://hits.dwyl.io/asetalias/asetalias.github.io) [![License](https://img.shields.io/github/license/asetalias/blog.svg?style=flat-square)](https://github.com/asetalias/asetalias.github.io/blob/master/LICENSE) 
```

Reply:
Copied it from the blog README, didn't give it a second look.